config/servicebus/config.go

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2022 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package servicebus
+
+import (
+	"github.com/upbound/upjet/pkg/config"
+)
+
+// Configure configures security group
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("azurerm_servicebus_namespace", func(r *config.Resource) {
+		r.UseAsync = true
+		// Issues on using Namespace as Kind in servicebus
+		// https://github.com/crossplane/terrajet/issues/234
+		// https://github.com/kubernetes/kubernetes/pull/108382
+		r.Kind = "ServicebusNamespace"
```

Comment:
What about using `ServiceBusNamespace` as the Azure provider is named as `Microsoft.ServiceBus`?:
```suggestion
		r.Kind = "ServiceBusNamespace"
```

Or, maybe:
```suggestion
		r.Kind = "NS"
```
, so that in contexts where the API Group and Kind are used, we would have something like `NS.servicebus` instead of `ServiceBusNamespace.servicebus`.

But it looks like previously we ended up using `EventHubNamespace` in a similar situation:
https://github.com/crossplane-contrib/provider-jet-azure/blob/9e1d60a75289e96b6b01ce99db6914080b508ff5/config/eventhub/config.go#L29

So, I would suggest the kind name to be `ServiceBusNamespace` to be consistent in naming.